Source/relay/Guide/Items of the Month/Better Shrooms and Gardens.ash

```diff
@@ -27,19 +27,60 @@ void IOTMBetterShroomsAndGardensGenerateResource(ChecklistEntry [int] resource_e
 
         return ChecklistSubentryMake(main_title, subtitle, description);
     }
+	
+	ChecklistSubentry getMushroomState() {
+		int mushroomLevel = get_property_int("mushroomGardenCropLevel");
+		int expectedFilets = MIN(3, mushroomLevel)*3;
+		int expectedSlabs = clampi(mushroomLevel - 3, 0, 2);
+		
+		// Title
+		string main_title = "Upkeep your Mushroom";
+		
+		// Subtitle
+		string subtitle = "";
+		
+		// Entries
+		string [int] description;
+		
+		string [int] shroomYield;
+		if (!get_property_boolean("_mushroomGardenVisited")) {
+			description.listAppend("Mushroom is at tier " + mushroomLevel);
+			shroomYield.listAppend(expectedFilets + " filets / 9");
+			if (mushroomLevel > 3) {
+				shroomYield.listAppend( pluralise(expectedSlabs,"slab","slabs") + " / 2" );
+			} else {
+				shroomYield.listAppend("+1 Slab at tier 4 & 5");
+			}
+			if (mushroomLevel > 10) {
+				shroomYield.listAppend("A mushroom house");
+				description.listAppend(HTMLGenerateSpanOfClass("No reason to wait any longer", "r_bold"));
+			} else {
+				shroomYield.listAppend("House at tier 11");
+			}
+			
+			description.listAppend("Will give:" + HTMLGenerateIndentedText(shroomYield));
+		}
+		
+		return ChecklistSubentryMake(main_title, subtitle, description);
+	}
```

Comment:
I find the wording on this tile a little confusing. Can we do something like:

--------------
**Upkeep your mushroom garden**

Currently at growth level 3. Will yield 9 mushroom filets if picked.
Could wait for higher yields:

-  Level 4: 9 mushroom filets, 1 mushroom slab
-  Level 5: 9 mushroom filets, 2 mushroom slabs
-  Level 11: 9 mushroom filets, 2 mushroom slabs, 1 house-sized mushroom
--------------